### DISCUSSION TITLE
Plot different grains in different colors

### DISCUSSION THREAD

**Discussion by NikitaSakharov** [mtex-toolbox/mtex#862] [Ask Anything] [answered]
What I'm trying to do is to color some grains with one color and others with another color. 
Unfortunately this does not work, the map I get is a monocromatic one.
Here is my code

mergedGrains.color = str2rgb('Blue')
grains.color = str2rgb('Yellow')
plot(mergedGrains)
hold on
plot(grains(nonTwinId))
hold off

![untitled](https://user-images.githubusercontent.com/62562246/111634876-f9add600-8807-11eb-8131-87a7ec0cc83b.jpg)

**Comment by ralfHielscher** [comment, marked as answer]
Hi Nikita,

this is not meant to work. The correct syntax is
```matlab
plot(grains,'facecolor','red')
```
Ralf.

**Comment by zmichels** [comment]
Try maybe:

plot(grains, ‘FaceColor’, ‘b’)

**Comment by NikitaSakharov** [comment]
By the way, mergedGrains and grains(nonTwinId) are different 'structures'. 

Here is example from which you can understand which of the grains are 'mergedGrains' and which of them are 'grains(nonTwinId) '
If we plot them using this code

plot(grains, grains.meanOrientation)
hold on
plot(grains(nonTwinId).boundary, 'linecolor','black','linewidth',2)
hold off

we get the following image 

![untitled](https://user-images.githubusercontent.com/62562246/111636021-26162200-8809-11eb-8761-7598ad06b973.jpg)

**Reply by NikitaSakharov** [reply]
thanks a lot! it works now
